edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -350,13 +354,32 @@ impl ModuleRuntime for DockerModuleRuntime {
                     module.config().image()
                 );
 
-                let create_options = create_options
+                let mut create_options = create_options
                     .with_image(module.config().image().to_string())
                     .with_env(merged_env)
                     .with_labels(labels);
 
-                // Here we don't add the container to the iot edge docker network as the edge-agent is expected to do that.
-                // It contains the logic to add a container to the iot edge network only if a network is not already specified.
+                // In the bootstrap scenario (edge runtime module starting up for the first time),
```

Comment:
"edge runtime" -> "edge agent" ? Or is calling it "runtime" intentional?

Reply:
I suspect it was intentional, although I tend to think of the runtime as the collection of all system modules (agent + hub). Anyway, elsewhere in the code it was referring to the edge runtime module, so I kept that here. I can change it to edge agent.

Reply:
Just noticed the comment I replaced referred to edge agent not runtime, so yeah, I'll change it.